public/locales/en/translation.json

```diff
@@ -41,6 +42,14 @@
     "siteMap": "Site Map",
     "copyright": "Copyright © 2020 State of California"
   },
+  "certification-header1": "What is certification?",
+  "certification-p1": "Certification is the required process of updating the EDD every two weeks with your unemployment status with basic eligibility information:",
+  "certification-list1-item1": "Are you physically able to work?",
+  "certification-list1-item2": "Are you available for work?",
+  "certification-list1-item3": "Are you ready and willing to accept work immediately?",
+  "certification-list1-item4": "Did you refuse work",
```

Comment:
Should this line end with a question mark?

Reply:
That seems extremely likely! Will add that for now, and ping @nguyen-tina in case this is somehow intentional